clients/cmd/logstash/lib/logstash/outputs/loki.rb

```diff
@@ -205,6 +205,7 @@ def is_batch_expired
   public
   def receive(event)
     @entries << Entry.new(event, @message_field, @include_fields, @metadata_fields)
+    @tenant_id = event.sprintf(@tenant_id)
```

Comment:
Would you mind adding a test for this please?

Reply:
Hey, this adds only string interpolation support for the tenant_id field. I did the test as documented in the readme. Where do I need to add the tests?

Reply:
I guess at least it should have `if @tenant_id` but I'm wondering wouldn't this override the global variable tenant_id on the first pass, so basically only getting the tenant id from the first message then using the same all the time ?

Reply:
I could be wrong but I think you want to change the Entry to accept a tenant ID.

Then you'll need a use a different batch for each tenantID.

May be I'm not fully understanding this.